### PR TITLE
chore(pacakge): update to ionic 3.5

### PR DESCRIPTION
**Pull request by danielsogl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,27 +13,27 @@
...
+    "@angular/core": "4.1.3",
+    "@angular/forms": "4.1.3",
+    "@angular/http": "4.1.3",
+    "@angular/platform-browser": "4.1.3",
+    "@angular/platform-browser-dynamic": "4.1.3",
     "@ionic-native/core": "3.12.1",
     "@ionic-native/splash-screen": "3.12.1",
     "@ionic-native/status-bar": "3.12.1",
     "@ionic/storage": "2.0.1",
     "ionic-angular": "3.3.0",
```
Comment (pwespi) [inline package.json]: You forgot ionic-angular itself... ;-)

Comment (danielsogl) [inline package.json]: Fail :D Thanks for pointing me out!

**Comment by gianpaj** [comment]
is anybody maintaining this repo?

**Comment by leifwells** [comment]
Yes, I am here. I've been waiting for the dust to settle with the Ionic folks and their updates to the framework, CLI and app-scripts before accepting this PR. Bear with me.

**Pull request merged**